NativeApp/include/Android/AndroidAppBase.hpp

```diff
@@ -89,7 +91,24 @@ class AndroidAppBase : public AppBase
     void UpdateFPS(float fFPS);
 
     bool initialized_resources_ = false;
-    bool has_focus_             = false;
+
+    enum
```

Comment:
I suggest give it a name (e.g. `APP_STATUS_FLAGS`) and use `DEFINE_FLAG_ENUM_OPERATORS(APP_STATUS_FLAGS)`

Reply:
`                eng->app_status_ |= APP_STATUS_FLAG_FOCUSED;`

Assigning to 'Diligent::AndroidAppBase::APP_STATUS_FLAGS' from incompatible type 'int'